### ISSUE TITLE
Display Fetched Data in Your App

### ISSUE THREAD

**Issue by bsachin** [sap-tutorials/sap-build-apps#651]
Tutorials: https://developers.sap.com/tutorials/appgyver-display-information.html
--------------------------

Write here how you think we can improve the tutorial ...
Point 1 : 
Context :
The data model could have been more simpler than having a deep nested structure of data, this specially makes looking for a data binding after scrolling through a long deep nested structure a very tedious task.

Improvements for BuildApps : Could allow a simple search feature or drill down and collapse option
Improvements in tutorial : Use a simpler REST API which has few fields..

Point 2:
Corrections in tutorial : Q for what was the text after scanning the barcode? 
Doesnt accept the value as read from the barcode via the app and also as per the barcode as the right ans 003120016034 is what is from the barcode and what gets shown in the app, but you dont accept it.

Point 3:
System/ build app set up
While creating a trial subaccount, we only get a SAP BAS subscribed.. required SAP Build Apps subscription is not done automatically ( even if we follow the step you have mentioned ).. We also have to first have an Cloud Identity Services subscribed and configured to then subscribe for SAP Build Apps, problem with the booster??

**Comment by thecodester** [comment]
1) I will let dev know of this request for search.

2) Which tutorial? In this tutorial the question is: "In the example of above of scanned food item (cranberry sauce), what is the calorific content displayed?" Is that the question you see? If so, what answer are you giving?

3)

**Comment by amshetgar** [comment]
> 2. Which tutorial? In this tutorial the question is: "In the example of above of scanned food item (cranberry sauce), what is the calorific content displayed?" Is that the question you see? If so, what answer are you giving? Yes i am giving the answer 816

**Comment by thecodester** [comment]
Yes, I see that the cranberry sauce has gotten more fattening ... I have updated the question
